python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -26,9 +26,10 @@
 from ray.autoscaler._private.util import ConcurrentCounter, validate_config, \
     with_head_node_ip, hash_launch_conf, hash_runtime_conf, \
     DEBUG_AUTOSCALING_STATUS, DEBUG_AUTOSCALING_ERROR
-from ray.ray_constants import AUTOSCALER_MAX_NUM_FAILURES, \
-    AUTOSCALER_MAX_LAUNCH_BATCH, AUTOSCALER_MAX_CONCURRENT_LAUNCHES, \
-    AUTOSCALER_UPDATE_INTERVAL_S, AUTOSCALER_HEARTBEAT_TIMEOUT_S
+from ray.autoscaler.ray_autoscaler_constants import \
```

Comment:
`ray.autoscaler.constants`? or else it's a bit redundant?

Reply:
`ray.autoscaler._private.constants`

Reply:
SGTM, that is far better.